### ISSUE TITLE
Unable to destroy Node Pool

### ISSUE THREAD

**Issue by alioSHA512** [aliyun/terraform-provider-alicloud#2881]
I've create a Node Pool `alicloud_cs_kubernetes_node_pool.temp-scaling` with 2 nodes inside.

Then I tried to delete the Node Pool completely. By comment out the resource `alicloud_cs_kubernetes_node_pool.temp-scaling`, I got `terraform plan` shows:

```shell
Terraform will perform the following actions:
  # alicloud_cs_kubernetes_node_pool.temp-scaling will be destroyed
  ...
Plan: 0 to add, 0 to change, 1 to destroy.
```

But `terraform apply` just keep printing  "Still destroying..." and nothing actually happened to the Node Pool.

### Terraform Version
Terraform v0.13.3
+ provider registry.terraform.io/hashicorp/alicloud v1.98.0
+ provider registry.terraform.io/hashicorp/external v1.2.0

### Affected Resource(s)
- alicloud_cs_kubernetes_node_pool

### Terraform Configuration Files
```hcl
resource "alicloud_cs_kubernetes_node_pool" "temp-scaling" {
  node_count = 2

  name = "temp-scaling"
  cluster_id = "--------------"
  vswitch_ids = ["vsw-------------------"]
  instance_types = ["ecs.c6.6xlarge"]
  key_name = var.key_name

  tags = {  ...  }
}
```

### Expected Behavior
Node Pool should be destroyed and the nodes should be (first drained then) removed from the K8s cluster.

### Actual Behavior
Nothing happened to the specifying Node Pool. Terraform apply just keep printing ` Still destroying...` forever. Nothing happened to the Node Pool, and it's not tainted in tfstate either.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. create a node pool via resource `alicloud_cs_kubernetes_node_pool`
2. delete the resource code block then `terraform apply`

### Important Factoids
My initial propose is to scale down the Node Pool, but got `node_count can not be less than before` after I modified `node_scale` to 0.

**Comment by xiaozhu36** [comment]
@demonwy Can you check it?

**Comment by alioSHA512** [comment]
I've found out that, the Node Pool must be empty (have 0 nodes in it) to be deleted.

But as I've stated, Node Pool can't be downscaled. We can upscale it, and then remove nodes, instead of set `node_count` to a smaller number.

My current workaround is bind a null_resource to the Node Pool `node_count` attr., and use the API `RemoveClusterNodes` via Python in the `local-exec` provisioner.

Looking forward to see if there's a recommended way to "downscale" a Node Pool.
